sirepo/auth/__init__.py

```diff
@@ -612,10 +612,13 @@ def _auth_state():
     return v
 
 def _create_roles_for_user(uid, method):
+    r = []
     if not (pkconfig.channel_in('dev') and method == METHOD_GUEST):
-        return
-
-    auth_db.UserRole.add_roles(uid, get_all_roles())
+        if not sirepo.template.is_sim_type('jupyterhublogin'):
+            return
+        # Every user gets the jupyterhub role initially
+        r.append(role_for_sim_type('jupyterhublogin'))
```

Comment:
only add the role if not a guest dev user? is this for testing? probably want to documnt

Reply:
6044c7cb. Is this clearer?

Reply:
Yes, but I get so confused with the fact that it is still showing the old code above....